.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # TODO: hoist to 10.x when we bump the version supported in #5931
+        node-version: [ 8.x ]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
```

Comment:
You might want to consider using `v2` here instead. https://github.com/actions/setup-node#v2-beta
I'm using it on all the Yari actions and it's working fine there. 

It's your call but perhaps now that this stuff is fresh in your mind, it might worth aiming for the bleeding edge so the stuff will work longer.

Reply:
Nice to hear that it's working fine. I'll switch it to v2. 👍